### ISSUE TITLE
[A11y] Homepage | About Us | Take-Down Policy

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3267]
### Operating System

MacOS

### Browser

Safari

### Affected page

Homepage

### Admin user?

No

### Issue Description

Link: https://oregondigital.org/takedown?locale=en

2 Error, see below:

<img width="767" alt="Image" src="https://github.com/user-attachments/assets/319850ac-dd12-4e1d-a3a9-3d35710886f0" />

### Suggested Fix

Fix ONE of the following:

The link has insufficient color contrast of 1.08:1 with the surrounding text. (Minimum contrast is 3:1, link text: #073c5c, surrounding text: #333333)

The link has no styling (such as underline) to distinguish it from the surrounding text

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Using Chrome Insights, revalidate the page for the error.
